### PR TITLE
test(integ): add integration tests for nlb support

### PR DESCRIPTION
**Pull request by Lou1415926**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
+//go:build integration || localintegration || temporary
```
Comment (iamhopaul123) [inline internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go:1]: Does it make more sense for the file name to be `nlb_web_service_integration_test`?

File: internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+			svcParamsPath: "svc-nlb-dev.params.json",
+		},
+		"prod env": {
+			envName:       "prod",
+			svcStackPath:  "svc-nlb-prod.stack.yml",
+			svcParamsPath: "svc-nlb-prod.params.json",
+		},
+	}
+	val, exist := os.LookupEnv("TAG")
+	require.NoError(t, os.Setenv("TAG", "cicdtest"))
```
Comment (iamhopaul123) [inline internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go:56]: "Setenv" instead of "SetEnv"? Amazing 🧐

**Review by iamhopaul123 — APPROVED** [review]

**Comment by iamhopaul123** [comment]
Also maybe rename the title and commit to be "test(integ): add integration tests for nlb support"?

File: internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go (hunk 1 above)
Comment (Lou1415926) [reply internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go:1]: I was thinking this as well but wanted them to be grouped with the other `lb_` tests (you know how editor lists the files alphabetically). But I'm fine with either!

File: internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go (hunk 2 above)
Comment (Lou1415926) [reply internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go:56]: i knowwww the whole [`os`](https://pkg.go.dev/os) package's function naming feels like a huge "I gave up"

**Review by efekarakus — APPROVED** [review]

**Pull request merged**